backend/logic/order/index.js

```diff
@@ -541,7 +541,9 @@ async function updateInventoryData(
 
   const quantModifier = increment ? 1 : -1
 
-  const cartItems = get(cartData, 'items', [])
+  const cartItems = get(cartData, 'items', []).filter(
```

Comment:
nit: add a comment explaining we filter out items for which the inventory is managed externally, such as Printful items.